sdk/clientcore/core/src/main/java/io/clientcore/core/instrumentation/logging/LoggingEvent.java

```diff
@@ -251,44 +262,30 @@ public void log(String message) {
      */
     public void log(Supplier<String> message) {
         if (this.isEnabled) {
-            logger.performLogging(level, getMessageWithContext(message.get()), null);
+            performLogging(message.get());
         }
     }
 
     /**
-     * Logs throwable annotated with context.
+     * Logs exception.
      *
      * @param throwable {@link Throwable} for the message.
      * @param <T> Type of the Throwable being logged.
      * @return The passed {@link Throwable}.
+     * @throws NullPointerException if {@code throwable} is null.
      */
-    public <T extends Throwable> T log(T throwable) {
-        return log(null, throwable);
+    public <T extends Throwable> T logThrowable(T throwable) {
```

Comment:
Do we want both `setThrowable` and `logThrowable`? If they are different exceptions, would it log both?